### PR TITLE
Figure out the InfluxDB version from /ping and show it on the admin page

### PR DESCRIPTION
**Pull request by alexiri**
I noticed the InfluxDB version at the bottom of the admin page is hardcoded (currently to 0.9.3) and it hasn't been updated to 0.9.4. #4117 updates the version, but here's a patch to dynamically get the version from /ping instead.

### CODE REVIEW

**Comment by corylanou** [comment]
Nice. +1. /cc @toddboom

**Comment by toddboom** [comment]
I like this. I'm wondering if it makes sense to have both the client version (possibly still hardcoded) and the server version (pulled from the ping), just to help with debugging in the future. What do you think?

**Comment by alexiri** [comment]
I'm not sure I see the usefulness of that, I wasn't aware that the admin page had a "client" version different from the server. I guess it could make sense if you were connecting from one server's admin page to a different server. Still, keeping a hardcoded version number that must be updated with each release negates a bit the spirit of this change, which was to hardcode fewer things. :)

By the way, the version number could also be extracted from the "SHOW DATABASES" query which runs at startup anyway.

**Comment by otoolep** [comment]
SHOW DIAGNOSTICS will return full build information.

**Comment by alexiri** [comment]
Yes, I meant that calling "/query?q=SHOW+DATABASES" would also return the header with the version that I'm using in my patch.

**Comment by otoolep** [comment]
Yeah, that query will work. The nice thing about `SHOW DIAGNOSTICS` is that the commit and branch of the build will also be returned. That richer information may be useful.

**Comment by simi** [comment]
It will be great to add `statik` compilation into build steps and hardcode admin version in the build process. I'll try to inspect this.

**Comment by corylanou** [comment]
@toddboom can you review this.  I'm fine with it if we rebase.  Thoughts?

File: shared/admin/index.html [hunk 1]
```diff
@@ -160,7 +160,7 @@ <h3 class="panel-title">Connection Settings</h3>
     <!-- /.container -->
     <div id="footer">
       <div class="container">
-          <p class="text-muted text-right credit"><b>InfluxDB</b> v0.9.3</p>
+          <p class="text-muted text-right credit"><b>InfluxDB</b> <span class="influxdb-version"></span></p>
```
Comment (simi) [inline shared/admin/index.html]: I think id can be used here instead of class to match another tag lookups in javascript.

**Comment by toddboom** [comment]
@corylanou agreed. i added one change to split out the version numbers, rebased, and merged. thanks @alexiri!

**Pull request merged**